site/content/en/docs/manual/advanced/analytics-and-monitoring/consensus.md

```diff
@@ -0,0 +1,179 @@
+merge annotations automatically to obtain more reliable annotations.
+
+> **Note** that consensus merging only supports
+> 2d tasks. It supports all the annotation types except 2d cuboids.
+
+> **Note** that consensus merging is currently available for tasks and jobs.
+
+CVAT has the following features related to consensus-based annotation:
+- Creation of **consensus replica** jobs for regular annotation jobs in a task
+- Automatic merging for annotations inside **consensus replica** jobs
+
+## Basics
+
+If you want to improve quality of your annotations, there are several widespread ways
+to achieve this. One of the methods is called _consensus-based annotation_ or just _consensus_.
+In this method, the same data is annotated several times. Once there are several different
+annotations ("opinions") for the same objects, they can be merged in order to obtain
+annotations of higher quality.
+
+Let's consider an explanatory example. Imagine there is a group of people,
+and you are asking them whether something is true or not. There are several
+possible combinations of votes, including all the votes only for one of the options.
+Typically, if you don't know the correct answer, you'd probably simply trust
+the most voted option. This is called _majority voting_ - and it requires
+such majority to exist, so in general, it's desirable to have an odd number
+of people in the group. If there is an even number of votes for both options,
+you don't have enough information to prefer one of the options to the other.
+With larger groups the method becomes less sensitive to this requirement,
+as when the voters are independent and the question is meaningful,
+the answers are less likely to separate evenly between the possible options.
+
+Returning back to datasets, it works very similar to the example above. Each image
+is annotated several times, typically by different persons, and then the resulting
+annotations are compared and merged using majority voting or some other strategy.
+
+Datasets, typically, have a big number of images and objects. This method of annotation,
+requires several different annotations for the dataset, so it is expected to have
+several times of the annotation costs compared to the simple single-annotation approach,
+while dataset annotation is not cheap. The question arises how to apply this method efficiently.
+One application for this approach that can be recommended is Ground Truth annotation.
+This type of annotations typically requires especially high quality annotations and
+includes a small portion of the whole dataset images (e.g. 3%).
+If we apply 3- or 5-fold _consensus_ to GT annotations, we can get reliable GT annotations
+for 10-15% full dataset annotation cost. Now, this idea actually looks interesting. Once
+you have such a reliable GT dataset, it can be used for the remaining dataset validation,
+as regular.
+
+## Consensus replica jobs
+
+A **Consensus Replica job** (replica) is a way in CVAT to represent one of
```

Comment:
```suggestion
A **Consensus Replica job** (replica) is CVAT's way of representing one of
```

Reply:
I don't like using this possessive, it hurts pronunciation. Updated

Reply:
The main reason I wanted to change this is because it's not just _a_ way, but _the_ way.